spec/presenters/search_reference_presenter_spec.rb

```diff
@@ -19,6 +19,13 @@
   end
 
   describe '#to_s' do
-    it { expect(presented.to_s).to eq('Tomatoes') }
+    it 'returns the correct string' do
+      expect(presented.to_s).to eq('Tomatoes')
+    end
+
+    it 'capitalizes the first letter of the string and leaves the rest as is' do
+      presented.title = 'semi-skimmed milk, not powdered'
```

Comment:
just a little thing here,  
presented should be initialised somehow instead of direct assignment here.